lib/gauge_wilson_flow.cu

```diff
@@ -17,11 +17,14 @@ namespace quda {
     const QudaGaugeSmearType wflow_type;
     const WFlowStepType step_type;
 
-    bool tuneSharedBytes() const { return false; }
     unsigned int minThreads() const { return in.LocalVolumeCB(); }
-    unsigned int maxBlockSize(const TuneParam &) const { return 32; }
-    int blockStep() const { return 8; }
-    int blockMin() const { return 8; }
+    unsigned int maxSharedBytesPerBlock() const { return maxDynamicSharedBytesPerBlock(); }
```

Comment:
Should this be dependent on `wflow_type == QUDA_GAUGE_SMEAR_SYMANZIK_FLOW` as well?

Reply:
b4317464f0a7be097777a50d6afd723b964b9e4e